lib/fog/openstack/models/volume/snapshot.rb

```diff
@@ -0,0 +1,32 @@
+require 'fog/openstack/models/model'
+
+module Fog
+  module Volume
+    class OpenStack
+      class Snapshot < Fog::OpenStack::Model
+        identity :id
+
+        attribute :display_name
+        attribute :display_description
```

Comment:
v1 cinder is already deprecated and v2 uses name and description over display_name and display_description.  It'd be nice if these two at least were aliased.